projects/online/online/main.py

```diff
@@ -27,6 +31,52 @@
 UPDATE_SIZE = 1
 
 
+def _event_processing_worker(queue: Queue):
```

Comment:
let's move these to a `mp.py` file. Can also just generalize this 

```python
def process_queue(queue: Queue, f: Callable, sleep: float = 1e-3):
    while True:
        try:
            args = queue.get()
            if args[0] is None:
                continue
            f(*args)
        except Empty:
            time.sleep(sleep)
```

And then setup another function

```python
def initialize_queue_processor(f: Callable) -> Queue:
    queue = Queue()
    queue.put((None,))
    process = Process(
        target=process_queue(f), args=(queue,)
    )
    process.start()
    return queue 
```

So that these can be re-used for buffer writer and event processor